portal/workspaces.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Workspaces
-description: Brief introduciton about Fiddler Jam workspaces 
+description: Brief introduction about Fiddler Jam workspaces 
```

Comment:
Try explaining in one sentence what workspaces are, or what benefits they bring, so that keywords are added to the meta description.